products.php

```diff
@@ -70,18 +44,488 @@
         'health_license_number' => '50/16098',
         'image'                 => 3,
         'is_public'             => 1,
-        'created_by'            => 1,
+        'created_by'            => 
     ],  [
-        'id'                    => 1,
-        'name'                  => '',
-        'alias'                 => '',
-        'description'           => '',
+        'id'                    => 4,
+        'name'                  => 'آبمیوه نکتار انبه',
+        'alias'                 => 'fruit juice mango nectar',
+        'description'           => 'آبمیوه   پوره طبعی انبه',
         'brand_id'              => 1,
         'sku'                   => 'SKU001',
-        'barcode'               => '',
+        'barcode'               => '6261111109769',
+        'category_id'           => 2,
+        'health_license_number' => '39/ظ/1113',
+        'image'                 => 4,
+        'is_public'             => 1,
+        'created_by'            => 1,
+    ],
+
+     [
+        'id'                    => 5,
+        'name'                  => 'آبمیوه طبیعی هلو',
+        'alias'                 => ' fruit nectar peach'
```

Comment:
```suggestion
        'alias'                 => 'fruit nectar peach'
```
چرا فضای خالی اضافی دارد؟